contracts/BatchExchangeViewer.sol

```diff
@@ -22,10 +23,10 @@ contract BatchExchangeViewer {
 
     /** @dev Queries the orderbook for the auction that is still accepting orders
      *  @param tokenFilter all returned order will have buy *and* sell token from this list (leave empty for "no filter")
-     *  @return encoded bytes representing orders
+     *  @return encoded bytes representing orders, maxed at 5000 elements
      */
     function getOpenOrderBook(address[] memory tokenFilter) public view returns (bytes memory) {
-        (bytes memory elements, , ) = getOpenOrderBookPaginated(tokenFilter, address(0), 0, uint16(-1));
+        (bytes memory elements, , ) = getOpenOrderBookPaginated(tokenFilter, address(0), 0, LARGE_PAGE_SIZE);
```

Comment:
Should this method fail if the page size is exceeded? I can imagine subtle issues happening when the orderbook exceeds 5000 open orders and the returned data does not represent the entire orderbook.